CMP1903M Assessment 1 Base Code/Analyse.cs

```diff
@@ -1,36 +1,63 @@
-﻿using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Text;
-using System.Threading.Tasks;
-
-namespace CMP1903M_Assessment_1_Base_Code
-{
-    public class Analyse
-    {
-        //Handles the analysis of text
-
-        //Method: analyseText
-        //Arguments: string
-        //Returns: list of integers
-        //Calculates and returns an analysis of the text
-        public List<int> analyseText(string input)
-        {
-            //List of integers to hold the first five measurements:
-            //1. Number of sentences
-            //2. Number of vowels
-            //3. Number of consonants
-            //4. Number of upper case letters
-            //5. Number of lower case letters
-            List<int> values = new List<int>();
-            //Initialise all the values in the list to '0'
-            for(int i = 0; i<5; i++)
-            {
-                values.Add(0);
-            }
-
-
-            return values;
-        }
-    }
-}
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using static CMP1903M_Assessment_1_Base_Code.Report;
+
+namespace CMP1903M_Assessment_1_Base_Code
+{
+    
+    public class Analyse
+    {
+        //Method: analyseText
+        //Arguments: string
+        //Returns: list of integers
+        //Calculates and returns an analysis of the text
+        public static List<int> analyseText(string input)
+        {
+            //list of integers to hold the first five measurements:
+
+            //1. number of sentences
+            int sentenceCount = input.Split('.').Length - 1;
+
+            //2. number of characters
+            int charCount = input.Count(c => "abcdefghijklmnopqrstuvwxyz".Contains(Char.ToLower(c)));

```

Comment:
An interesting method of doing this, never thought about doing it this way. Looks goos 👍